services/project-grafana-logging/6.60.1/defaults/cm.yaml

```
@@ -7,9 +7,8 @@ data:
   values.yaml: |
     ---
     priorityClassName: "dkp-critical-priority"
-    # pinning grafana to 8.5 since it appears that v9+ is causing the kubernetes audit dashboard to crash
     image:
-      tag: 8.5.26
+      tag: 9.5.13
```

```suggestion
      tag: 8.5.26
```

we need to use an earlier version of grafana for our logging instances https://github.com/mesosphere/kommander-applications/pull/1760